api/handler/v1beta1/adapter.go

```diff
@@ -167,6 +181,19 @@ func (adapt *Adapter) ToJobProto(spec models.JobSpec) (*pb.JobSpecification, err
 			Config:   notify.Config,
 		})
 	}
+	//prep external dependencies for proto
+	//var externalDependenciesProto *pb.ExternalDependency
+	externalDependenciesProto := &pb.ExternalDependency{
```

Comment:
can you update this as discussed by keeping these under dependencies itself

Reply:
addressed under latest commit